runtime/src/snapshot_utils.rs

```diff
@@ -413,7 +414,7 @@ pub enum AddBankSnapshotError {
     #[error("bank snapshot dir already exists: {0}")]
     SnapshotDirAlreadyExists(PathBuf),
 
-    #[error("failed to create snapshot dir: {0}")]
+    #[error("failed to create snapshot dir: {0}: {}", .0.source().unwrap())]
```

Comment:
why are these unwraps safe?

Reply:
but the api returns `Option<T>`. a comment in our code won't stop upstream from starting to return `None` in the future.

Reply:
For me, I'd rather avoid backporting something with these unwraps, same concerns as Trent. Better error messaging is great, but don't think it's a critical fix that needs immediate backporting - especially given the possibility of unwrapping a none.